master/reference/kube-controllers/configuration.md

```diff
@@ -80,6 +79,12 @@ To enable the node controller, perform the following two steps.
 
 This controller is only valid when using etcd as the {{site.prodname}} datastore.
 
+Set `SYNC_NODE_LABELS` to true (enabled by default) to ensure that labels on
+Kubernetes node resources remain in-sync with labels on the corresponding Calico
```

Comment:
I understand about the node resource but we should still do `s/Calico/{{site.prodname}}`